tfhe/docs/how_to/rust_configuration.md

```diff
@@ -0,0 +1,54 @@
+# Using TFHE-rs with nightly toolchain.
```

Comment:
could be good to still talk about a default toolchain though I understand why we would recommend a nightly toolchain to be sure more advanced stuff works well

Reply:
Talking only about nightly might make user think that we do not support stable rust.

Only certain features/things require nightly (eg. avx, C API)